crates/indexer/src/geyser/instructions/buy.rs

```diff
@@ -0,0 +1,174 @@
+use borsh::BorshDeserialize;
+use indexer_core::{
+    db::{
+        custom_types::OfferEventLifecycleEnum,
+        insert_into,
+        models::{BuyInstruction, FeedEventWallet, Offer, OfferEvent},
+        select,
+        tables::{
+            buy_instructions, current_metadata_owners, feed_event_wallets, feed_events, metadatas,
+            offer_events, offers,
+        },
+    },
+    uuid::Uuid,
+};
+
+use super::Client;
+use crate::prelude::*;
+
+#[derive(BorshDeserialize, Debug, Clone)]
+pub struct InstructionParameters {
+    trade_state_bump: u8,
+    escrow_payment_bump: u8,
+    buyer_price: u64,
+    token_size: u64,
+}
```

Comment:
anchor should be exposing this struct as well. not a huge deal but does help keep the source the contract.

https://github.com/metaplex-foundation/metaplex-program-library/blob/master/auction-house/program/src/receipt/mod.rs#L305

Reply:
Oh I missed this so will change it back to anchor one